subprojects/docs/src/docs/release/notes.md

```diff
@@ -36,6 +36,15 @@ The following are the newly deprecated items in this Gradle release. If you have
 
 ## Potential breaking changes
 
+### Changes in the caching of missing versions
+
+Previously, Gradle would refresh the version list when dynamic version cache expires for repositories that did not contain a version at all.
+From now on, Gradle [will cache forever that a `group:name` was absent](https://github.com/gradle/gradle/issues/4436) from a repository.
+This will impact positively dependency resolution when multiple repositories are defined and dynamic versions are used.
```

Comment:
"This will impact positively dependency resolution" -> "This has a positive performance impact on dependency resolution"

Reply:
Done